web/react/components/sidebar.jsx

```diff
@@ -49,8 +52,44 @@ export default class Sidebar extends React.Component {
         state.loadingDMChannel = -1;
         state.windowWidth = Utils.windowWidth();
 
+        this.unreadCount = this.setUnreadCount();
```

Comment:
The variable name doesn't make it obvious that it's a map of channel id to an unread, it just sounds like a single integer value. Maybe rename it `unreadChannelCounts` or something a little more descriptive?